docs/fileformat-prose.md

```diff
@@ -209,12 +292,38 @@ pca/variance_ratio <HDF5 dataset "variance_ratio": shape (50,), type "<f4">
 Zero dimensional arrays are used for scalar values (i.e. single values like strings, numbers or booleans).
 These should only occur inside of `uns`, and are commonly saved parameters:
 
+`````{tab-set}
+
+````{tab-item} HDF5
+:sync: hdf5
+
 ```python
->>> f["uns/neighbors/params"].visititems(print)
+>>> store["uns/neighbors/params"].visititems(print)
 method <HDF5 dataset "method": shape (), type "|O">
 metric <HDF5 dataset "metric": shape (), type "|O">
 n_neighbors <HDF5 dataset "n_neighbors": shape (), type "<i8">
->>> f["uns/neighbors/params/metric"][()]
+random_state <HDF5 dataset "random_state": shape (), type "<i8">
+```
+
+````
+
+````{tab-item} Zarr
+:sync: zarr
+
+```python
+>>> store["uns/neighbors/params"].visititems(print)
+method <zarr.core.Array '/uns/neighbors/params/method' () <U4 read-only>
```

Comment:
for HDF5 the types for method and metric are |0 and |0, while for Zarr are different. Is this correct?

Reply:
From the specification below:

> In zarr, scalar strings MUST be stored as a fixed length unicode dtype

IIRC think this was an issue with how zarr v2 does UTF8.